Wox/Storage/TopMostRecord.cs

```diff
@@ -1,47 +1,52 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
+using System.Runtime.InteropServices;
```

Comment:
Quick reminder for me again which call requires the InteropServices library?

Reply:
Its not used, removed.